### PR TITLE
Add NEP 29 authors to spec0

### PR DESCRIPTION
**Pull request by munkm**
This PR adds authors who originally contributed to NEP 29 to spec0. The authors I already added I have verbal consent from or have already contributed to NEP 0 at the developer summit. 

@tacaswell @rgommers @amueller and @ellisonbg would you like to be added as authors to this SPEC as well?

### CODE REVIEW

File: spec-0000/index.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 title: "SPEC 0 — Minimum Supported Versions"
 date: 2020-12-17
 author:
+  - "Brigitta Sipőcz <brigitta.sipocz@gmail.com>"
   - "Jarrod Millman <millman@berkeley.edu>"
+  - "Madicken Munk <madicken@berkeley.edu>"
+  - "Matt Haberland <mhaberla@calpoly.edu>"
+  - "Matthias Bussonnier <bussonniermatthias@gmail.com>"
```
Comment (tacaswell) [inline spec-0000/index.md:9]:
```suggestion
  - "Matthias Bussonnier <bussonniermatthias@gmail.com>"
  - "Thomas A Caswell <tcaswell@gmail.com>"
```

**Comment by amueller** [comment]
I don't think I'm an author, thank you though :)

**Comment by stefanv** [comment]
@amueller Are you sure?! https://numpy.org/neps/nep-0029-deprecation_policy.html

**Comment by amueller** [comment]
Huh, I stand corrected. I guess it makes sense to include me for consistency then? I have no strong opinion, though.

**Comment by rgommers** [comment]
Thanks for asking @munkm. Please include me as well (` Ralf Gommers <ralf.gommers@gmail.com>`).

**Comment by munkm** [comment]
Ok, I added your information Ralf. Andreas, do you have a preferred email contact?

**Comment by amueller** [comment]
`Andreas C. Mueller <andreas.mueller.ml@gmail.com>`

**Review by jarrodmillman — APPROVED** [review]

**Comment by stefanv** [comment]
Thanks, everyone.

**Pull request merged**